living_documentation_generator/generator.py

```diff
@@ -252,7 +254,14 @@ def _fetch_github_project_issues(self) -> dict[str, ProjectIssue]:
                         project_issue.repository_name,
                         project_issue.number,
                     )
-                    all_project_issues[key] = project_issue
+
+                    # If the key is unique, add the project issue to the dictionary
+                    if key not in all_project_issues:
+                        all_project_issues[key] = []
+                        all_project_issues[key].append(project_issue)
```

Comment:
```suggestion
                        all_project_issues[key] = [project_issue]
```

Reply:
Suggestion added in [6e9aa47](https://github.com/AbsaOSS/living-doc-generator/pull/39/commits/6e9aa47fc2aacf2162dfb50e1d66fa44869f8e8c).